Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -893,81 +916,79 @@ - (instancetype)initWithParentAdapter:(ALVungleMediationAdapter *)parentAdapter
+        extraInfo[@"ad_width"] = @(adSize.width);
+        extraInfo[@"ad_height"] = @(adSize.height);
     }
     
-    if ( [banner canPlayAd] )
-    {
-        [banner presentOn: self.parentAdapter.adViewContainer];
-    }
-    else
-    {
-        [self.parentAdapter log: @"Failed to load ad view ad: ad not ready"];
-        [self.delegate didFailToLoadAdViewAdWithError: MAAdapterError.adNotReady];
-    }
+    [self.delegate performSelector: @selector(didLoadAdForAdView:withExtraInfo:)
+                        withObject: bannerView
+                        withObject: extraInfo];
 }
 
-- (void)bannerAdDidFailToLoad:(VungleBanner *)banner withError:(NSError *)error
+- (void)bannerAdDidFail:(VungleBannerView *)bannerView withError:(NSError *)error
 {
     MAAdapterError *adapterError = [ALVungleMediationAdapter toMaxError: error isAdPresentError: NO];
     [self.parentAdapter log: @"AdView failed to load with error: %@", adapterError];
     [self.delegate didFailToLoadAdViewAdWithError: adapterError];
 }
 
-- (void)bannerAdWillPresent:(VungleBanner *)banner
+- (void)bannerAdWillPresent:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad will present %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad will present %@", bannerView.placementId];
 }
 
-- (void)bannerAdDidPresent:(VungleBanner *)banner
+- (void)bannerAdDidPresent:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad shown %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad shown %@", bannerView.placementId];
 }
 
-- (void)bannerAdDidTrackImpression:(VungleBanner *)banner
+- (void)bannerAdDidTrackImpression:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad impression tracked %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad impression tracked %@", bannerView.placementId];
     [self.delegate didDisplayAdViewAd];
 }
 
-- (void)bannerAdDidClick:(VungleBanner *)banner
+- (void)bannerAdDidClick:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad clicked %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad clicked %@", bannerView.placementId];
     [self.delegate didClickAdViewAd];
 }
 
-- (void)bannerAdWillLeaveApplication:(VungleBanner *)banner
+- (void)bannerAdWillLeaveApplication:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad will leave application %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad will leave application %@", bannerView.placementId];
 }
 
-- (void)bannerAdDidFailToPresent:(VungleBanner *)banner withError:(NSError *)error
+- (void)bannerAdWillClose:(VungleBannerView *)bannerView
 {
-    MAAdapterError *adapterError = [ALVungleMediationAdapter toMaxError: error isAdPresentError: YES];
-    [self.parentAdapter log: @"AdView ad failed to present with error: %@", adapterError];
-    [self.delegate didFailToDisplayAdViewAdWithError: adapterError];
```

Comment:
We will not notify any ad presenting error to the mediation, right? Thanks a lot. :)

Reply:
The callback method here has been changed from `bannerAdDidFailToPresent` to `bannerAdWillClose`.
We don't have `bannerAdDidFailToPresent` anymore with `VungleBannerView` class.

Reply:
Now, it's taken care at `bannerAdDidFail` callback.